bundle-workflow/src/run_integ_test.py

```diff
@@ -41,13 +41,14 @@ def pull_common_dependencies(work_dir, build_manifest):
             GitRepository(
                 component.repository,
                 component.commit_id,
-                os.path.join(work_dir, component.name),
+                os.path.join(work_dir, 'dependencies', component.name),
             )
 
 
 # TODO: replace with DependencyProvider - https://github.com/opensearch-project/opensearch-build/issues/283
 def sync_dependencies_to_maven_local(work_dir, manifest_build_ver):
-    os.chdir(work_dir + "/OpenSearch")
+    dependencies_dir = os.path.join(work_dir, 'dependencies')
+    os.chdir(dependencies_dir + "/OpenSearch")
```

Comment:
Change `run` to `check_output`, which incidentally supports `cwd=` for the directory in which it runs.

Reply:
+1